app/models/create_draft_edition.rb

```diff
@@ -39,6 +39,10 @@ def document
   def fill_out_new_edition
     document.increment!(:stale_lock_version)
     set_first_published_at
+    set_publisher_first_published_at
```

Comment:
Similar to in `update_existing_draft_edition.rb` I think we can simplify this with a loop over the attributes and just take them from the payload.